projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -418,17 +429,7 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     return depth > 0 && !node?.children ? -1 : 0;
   }
 
-  /**
-   * // Replace spaces with hyphens and convert to lowercase
-   */
-  getSanitizedTitle(title: string | undefined): string | null {
```

Comment:
While we may not be using the methods anymore, we can't really just remove them without deprecating them first.